src/dist/replication/replication.thrift

```diff
@@ -436,6 +436,33 @@ struct query_replica_info_response
     2:list<replica_info>  replicas;
 }
 
+struct disk_info
+{
+    // TODO(jiashuo1): figure out what the "tag" means and decide if it's necessary
+    1:string tag;
+    2:string full_dir;
+    3:i64 disk_capacity_mb;
+    4:i64 disk_available_mb;
+    // map<i32,i32> means map<app_id, replica_counts>
+    5:map<i32,i32> holding_primary_replica_counts;
+    6:map<i32,i32> holding_secondary_replica_counts;
+}
+
+// This request is sent from client to replica_server.
+struct query_disk_info_request
+{
+    1:dsn.rpc_address node;
+}
+
+// This response is recieved from client for replica_server.
```

Comment:
```suggestion
// This response is received from replica_server.
```
Actually no comment is OK... Because the comment on `query_disk_info_request` is clear enough.